src/genericComponents/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import styled from 'styled-components';
+// import { colors, fonts } from '../theme/styleVars';
+
+const ButtonPrimary = (props: any) => {
+  const { onClick, text, variant } = props;
```

Comment:
You probably also want to grab `...rest` so you can pass `{...rest}` down below in your button. This is in case any other props you haven't explicitly outlined here are added to a button in the wild.